backend/src/ml_space_lambda/emr/lambda_functions.py

```diff
@@ -158,7 +138,7 @@ def create(event, context):
                 },
             },
         ],
-        "Ec2KeyName": ec2_key_name,
+        "Ec2KeyName": "",
```

Comment:
It looks like the `cluster_config` we were reading from s3 before could possibly set this even though it defaulted to empty. Is this an ability to customers we aren't offering anymore?

Reply:
Asked Kyle and he said as far as he's aware nobody was using this. If it's something we still want to support I think we can just move it to a field in `constants.ts` and let it be a deployment time configuration.